pkg/cmd/projectCreate.go

```diff
@@ -489,6 +495,40 @@ func createTestProject(
 	return err
 }
 
+// createTestProject - creates a single project to contain tests which test a feature.
+func createTestManager(
+	fileGenerator *utils.FileGenerator,
+	packageName string,
+  manager string,
+	forceOverwrite bool,
+	useMaven bool,
+	useGradle bool,
+	isDevelopment bool,
+) error {
+
+	targetFolderPath := packageName + "/" + packageName + "." + manager
+	log.Printf("Creating tests project %s\n", targetFolderPath)
+
+	// Create the base test folder
+	err := fileGenerator.CreateFolder(targetFolderPath)
+	if err == nil {
+    if useMaven {
+			err = createTestFolderPom(fileGenerator, targetFolderPath, packageName, manager, forceOverwrite)
+		}
+
+		err = createTestFolderGradle(fileGenerator, targetFolderPath, packageName, manager, forceOverwrite, isDevelopment)
```

Comment:
We should only create the gradle file if useGradle is set, and that err is still null ?